### ISSUE TITLE
[Bug] SDK - `openbb.economy` shows functions for `macro_parameters`, but there is no `country` argument.

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#3139]
For example, `openbb.economy.inf()`

![Screen Shot 2022-10-27 at 3 20 54 PM](https://user-images.githubusercontent.com/85772166/198409142-27b66344-0db7-42fb-9925-9bc84a96924d.png)

or, `openbb.economy.cpi()`

**Comment by northern-64bit** [comment]
The `inf` and `cpi` commands use the alphavantage data source, which only has this data for the US. This data is not used for any commands in the terminal and can be regarded as legacy code, thus I don't know if it makes any sense to have this in the SDK due to better data source alternatives (example `FRED`). What do you think?

**Comment by deeleeramone** [comment]
Agreed, there is also `gdp`, 'gdpdc, `unemp`. Having these as part of the `menu` adds clutter.

**Comment by montezdesousa** [comment]
@northern-64bit do you need help on this?
